frontend-student/src/components/home/Home.jsx

```
@@ -22,34 +30,23 @@ export default function Home() {
+      {/* Image Carousel using Tailwind CSS */}
+      <div className="relative w-full h-[80vh] overflow-hidden">
+        <div
+          ref={carouselRef}
+          className="flex w-full h-full animate-carousel"
+        >
           {[image1, image2, image3].map((img, index) => (
-            <CarouselItem key={index} className="relative">
+            <div key={index} className="w-full flex-shrink-0">
               <img
                 src={img}
                 alt={`Image ${index + 1}`}
```

Biome (1.9.4)

[error] Avoid the words "image", "picture", or "photo" in img element alt text.

Screen readers announce img elements as "images", so it is not necessary to redeclare this in alternative text.

(lint/a11y/noRedundantAlt)